modules/ROOT/pages/storage.adoc

```diff
@@ -264,9 +264,13 @@ storage:
       with_mount_unit: true
 ----
 
-The root filesystem can also be moved to LUKS. In the case of the root filesystem the LUKS device must be backed by https://coreos.github.io/ignition/operator-notes/#clevis-based-devices[clevis]. There is simplified Butane config syntax for encrypting the root filesystem; for example:
+The root filesystem can also be moved to LUKS. In that case, the LUKS device must be pinned by https://coreos.github.io/ignition/operator-notes/#clevis-based-devices[Clevis]. There are two primary pin types available: TPM2 and Tang (or a combination of those using Shamir Secret Sharing).
 
-.Moving the root filesystem to LUKS
+WARNING: TPM2 pinning alone provides only limited protection. Make sure to understand its threat model before choosing between TPM2 and Tang pinning. For more information, see https://github.com/latchset/clevis/blob/master/src/pins/tpm2/clevis-encrypt-tpm2.1.adoc#threat-model[this section] of the Clevis TPM2 pin documentation.
```

Comment:
How about:
`NOTICE: TPM2 pinning just binds the encryption to presence of the machine.`

Reply:
Thank! I tweaked it a bit from your suggestion, but kept it as a `WARNING` to increase the chances users read that box before continuing.

Reply:
I'm OK to get this in as is.

But I also think `WARNING` is still too strong for this.  `IMPORTANT NOTICE:` ?

Reply:
Or how about `SECURITY: `?